src/daemon/dlt-daemon.c

```diff
@@ -243,6 +244,7 @@ int option_file_parser(DltDaemonLocal *daemon_local)
     daemon_local->flags.offlineLogstorageMaxCounter = UINT_MAX;
     daemon_local->flags.offlineLogstorageMaxCounterIdx = 0;
     daemon_local->flags.offlineLogstorageCacheSize = 30000; /* 30MB */
+    memset(daemon_local->flags.daemonFifoGroup, 0, sizeof(daemon_local->flags.daemonFifoGroup));
```

Comment:
As this is only for FIFO, please put the code inside #ifndef DLT_USE_UNIX_SOCKET_IPC. For example, this can go to line 259 before endif.